asserts/snap_asserts.go

```diff
@@ -171,18 +172,18 @@ func (snaprev *SnapRevision) SnapDigest() string {
 	return snaprev.Header("snap-digest")
 }
 
-// SnapRevision returns the revision of the snap-id assigned to this build.
-func (snaprev *SnapRevision) SnapRevision() uint64 {
-	return snaprev.snapRevision
+// SnapSize returns the size in bytes of the snap submitted to the store.
+func (snaprev *SnapRevision) SnapSize() uint64 {
+	return snaprev.snapSize
 }
 
-// SnapBuild returns the digest of the associated snap-build.
-func (snaprev *SnapRevision) SnapBuild() string {
```

Comment:
My apologies. As Samuele pointed out to me, the digest of the snap is still here, so my concern is sorted.